docs/en_US/BuiltinTuner.md

```diff
@@ -366,3 +366,45 @@ advisor:
     max_budget: 27
     eta: 3
 ```
+<br>
+
+<a name="GPTuner"></a>
+
+![](https://placehold.it/15/1589F0/000000?text=+) `GP Tuner`
+
+> Builtin Tuner Name: **GPTuner**
+
+Note that the only acceptable types of search space are `choice`, `quniform`, `uniform` and `randint`.
```

Comment:
why cannot support other type?

Reply:
`loguniform`, `qloguniform` added. Types like `normal` are not supported here since a limited bound is needed.